R/Chromatograms.R

```diff
@@ -159,4 +187,57 @@ setMethod("show", "Chromatograms",
               }
           })
 
+#' @rdname Chromatograms
+#'
+#' @note
+#' This needs to be discussed, if we want for example to be able to set a
+#' a backend to `ChromBackendMzR` we need to implement backendInitialize()
+#' better. = Support peaksData and chromData as arguments AND have a way to
+#' write .mzml files (which we do not have for chromatographic data).
+#'
+#' @importMethodsFrom ProtGenerics setBackend
+#'
+#' @exportMethod setBackend
+setMethod(
+    "setBackend", c("Chromatograms", "ChromBackend"),
+    function(object, backend, f = processingChunkFactor(object),
+             BPPARAM = SerialParam(), ...) {
+        backend_class <- class(object@backend)
+        BPPARAM <- backendBpparam(object@backend, BPPARAM)
+        BPPARAM <- backendBpparam(backend, BPPARAM)# what
```

Comment:
FYI: this is just to use the parallel setup supported by both backends, the current and the future one.